src/renderer.tsx

```diff
@@ -87,24 +101,26 @@ export const render = async (jsx: any, rootNode) => {
         // Append root node to a container
         appendInitialChild: (parentNode, childNode) => {
             console.log('appendInitialChild', parentNode, childNode);
-            appendToContainer(parentNode, childNode);
+            appendToContainer(groupsProcessor, parentNode, childNode);
         },
         appendChild: (parentNode, childNode) => {
             console.log('appendChild', parentNode, childNode);
-            appendToContainer(parentNode, childNode);
+            appendToContainer(groupsProcessor, parentNode, childNode);
```

Comment:
It can be implemented more FP-like. 😃

At line 11:
```javascript
const appendToContainerFactory = groupsProcessor => (parentNode, childNode) => {
```
At line 68:
```javascript
const groupsProcessor = new GroupsProcessor();
const appendToContainer = appendToContainerFactory(groupsProcessor);
```
At this line and others:
```javascript
appendToContainer(parentNode, childNode);
```

Reply:
@ilyalesik yeah, right. Replaced it